wfsim/core.py

```diff
@@ -491,7 +489,6 @@ class S2(Pulse):
 
     def __init__(self, config):
         super().__init__(config)
-        # TODO
         # This config is not set for the 1T fax config
         self.config.setdefault('s2_time_spread', 1)
```

Comment:
If we remove the #TODO, can we also remove these two lines?

Reply:
Both should be removed,. I did some github magic by accident causing them to reappear

Reply:
Thanks!